Models/ViewModel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Models;
+
+class ViewModel

```

Comment:
move it to separate directory in case of we follow MVC pattern
models are responsible for business logic 
controllers for connection model -> view
view must be responsible for representing of files